The files belonging to this database system will be owned by user "postgres".
This user must also own the server process.

The database cluster will be initialized with locale "en_US.utf8".
The default database encoding has accordingly been set to "UTF8".
The default text search configuration will be set to "english".

Data page checksums are disabled.

fixing permissions on existing directory /var/lib/postgresql/data/pgdata ... ok
creating subdirectories ... ok
selecting default max_connections ... 100
selecting default shared_buffers ... 128MB
selecting dynamic shared memory implementation ... posix
creating configuration files ... ok
running bootstrap script ... ok
performing post-bootstrap initialization ... ok

WARNING: enabling "trust" authentication for local connections
You can change this by editing pg_hba.conf or using the option -A, or
--auth-local and --auth-host, the next time you run initdb.
syncing data to disk ... ok

Success. You can now start the database server using:

    pg_ctl -D /var/lib/postgresql/data/pgdata -l logfile start

waiting for server to start....2025-02-14 09:00:46.484 UTC [46] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
2025-02-14 09:00:46.498 UTC [47] LOG:  database system was shut down at 2025-02-14 09:00:46 UTC
2025-02-14 09:00:46.502 UTC [46] LOG:  database system is ready to accept connections
 done
server started
CREATE DATABASE


/usr/local/bin/docker-entrypoint.sh: ignoring /docker-entrypoint-initdb.d/*

waiting for server to shut down....2025-02-14 09:00:46.848 UTC [46] LOG:  received fast shutdown request
2025-02-14 09:00:46.849 UTC [46] LOG:  aborting any active transactions
2025-02-14 09:00:46.850 UTC [46] LOG:  background worker "logical replication launcher" (PID 53) exited with exit code 1
2025-02-14 09:00:46.850 UTC [48] LOG:  shutting down
2025-02-14 09:00:46.866 UTC [46] LOG:  database system is shut down
 done
server stopped

PostgreSQL init process complete; ready for start up.

2025-02-14 09:00:46.955 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
2025-02-14 09:00:46.956 UTC [1] LOG:  listening on IPv6 address "::", port 5432
2025-02-14 09:00:46.957 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
2025-02-14 09:00:46.963 UTC [64] LOG:  database system was shut down at 2025-02-14 09:00:46 UTC
2025-02-14 09:00:46.967 UTC [1] LOG:  database system is ready to accept connections
2025-02-14 09:00:53.353 UTC [79] ERROR:  relation "_prisma_migrations" does not exist at character 126
2025-02-14 09:00:53.353 UTC [79] STATEMENT:  SELECT "id", "checksum", "finished_at", "migration_name", "logs", "rolled_back_at", "started_at", "applied_steps_count" FROM "_prisma_migrations" ORDER BY "started_at" ASC
2025-02-14 09:00:53.729 UTC [79] ERROR:  syntax error at or near "WITH" at character 89
2025-02-14 09:00:53.729 UTC [79] STATEMENT:  DROP DATABASE IF EXISTS "prisma_migrate_shadow_db_eff904b5-b1a8-4044-8395-cca25881105d" WITH (FORCE)
2025-02-14 09:00:54.302 UTC [79] ERROR:  syntax error at or near "WITH" at character 89
2025-02-14 09:00:54.302 UTC [79] STATEMENT:  DROP DATABASE IF EXISTS "prisma_migrate_shadow_db_b334fce0-28b2-4ba5-9199-e3d1f55a26f3" WITH (FORCE)
2025-02-14 09:00:54.843 UTC [79] ERROR:  syntax error at or near "WITH" at character 89
2025-02-14 09:00:54.843 UTC [79] STATEMENT:  DROP DATABASE IF EXISTS "prisma_migrate_shadow_db_f2ed05f8-6fce-4de4-889a-9aa19b22cf1e" WITH (FORCE)
2025-02-14 09:00:55.370 UTC [79] ERROR:  syntax error at or near "WITH" at character 89
2025-02-14 09:00:55.370 UTC [79] STATEMENT:  DROP DATABASE IF EXISTS "prisma_migrate_shadow_db_c6bcf72d-ba4c-43d3-888a-85c1fab8b8e4" WITH (FORCE)
